verdaccio.install.yaml

```diff
@@ -15,3 +15,4 @@ packages:
     publish: $all
     proxy: npmjs
 log: { type: stdout, format: pretty, level: http }
+max_body_size: 20mb
```

Comment:
The default `max_body_size` is 10mb. The tarball of `clerk-js` built by Rspack is over 15mb, so this was increased to prevent a 413 error during integration tests.